shared/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -153,6 +153,85 @@ namespace datadog::shared::nativeloader
                     return CORPROF_E_PROFILER_CANCEL_ACTIVATION;
                 }
             }
+
+            // If we weren't on the explicit include list, then try to filter out `dotnet build` etc.
+            // We don't want to instrument _build_ processes in dotnet by default, as they generally
+            // don't give useful information, add latency, and risk triggering bugs in the runtime,
+            // particularly around shutdown, like this one: https://github.com/dotnet/runtime/issues/55441
+            const auto command_line = GetCurrentProcessCommandLine();
+            const auto process_command_line = std::get<0>(command_line);
+            const auto command_line_arguments = std::get<1>(command_line);
```

Comment:
I think you can simplify the code
```suggestion
           const auto [command_line , process_command_line]  = GetCurrentProcessCommandLine();
```

Reply:
ooooh, pretty 😄